hack/verify-test-code.sh

```diff
@@ -51,6 +51,30 @@ do
     fi
 done
 
+errors_expect_be_bool=()
+for file in "${all_e2e_files[@]}"
+do
+    if grep -E "gomega\.Expect\(.*\)\.(To|NotTo)\(gomega\.(BeTrue|BeFalse)\(\)" "${file}" > /dev/null
+    then
+        errors_expect_be_bool+=( "${file}" )
+    fi
+done
+
+if [ ${#errors_expect_be_bool[@]} -ne 0 ]; then
+  {
+    echo "Errors:"
+    for err in "${errors_expect_be_bool[@]}"; do
+      echo "$err"
+    done
+    echo
+    echo 'The above files need to use framework.ExpectEqual(foo, true, explain) or '
+    echo 'framework.ExpectEqual(foo, false, explain) instead of '
+    echo 'gomega.Expect(foo).To(gomega.BeTrue(), explain) or gomega.Expect(foo).To(gomega.BeFalse(), explain)'
```

Comment:
can the message include examples like so?

`framework.ExpectEqual(foo, true|false, explain)`
`gomega.Expect(foo).To(gomega.BeTrue()|gomega.BeFalse(), explain`